handshake_client.go

```diff
@@ -423,6 +424,13 @@ func (hs *clientHandshakeState) doFullHandshake() error {
 			return err
 		}
 	}
+	c.useEMS = hs.serverHello.extendedMSSupported
```

Comment:
https://tools.ietf.org/html/rfc7627. The handshake hash does not include the client cert chain when the master secret is generated, only the ClientKeyExchange